README.md

```diff
@@ -4,7 +4,19 @@ Terraform module for [Azure Database for PostgreSQL - Flexible Server](https://d
 ## Example
 
 ```hcl
+provider "azurerm" {
+  features {}
+  skip_provider_registration = true
+  alias                      = "postgres_network"
+  subscription_id            = var.aks_subscription_id
+}
```

Comment:
is it worth creating separate code blocks for this do you think?

e.g.

provider.tf
```terraform
provider "azurerm" {
  features {}
  skip_provider_registration = true
  alias                      = "postgres_network"
  subscription_id            = var.aks_subscription_id
}
```

variables.tf
```terraform
variable "aks_subscription_id" {} # provided by the Jenkins library, ADO users will need to specify this
```

postgres.tf
```terraform
...
```

Reply:
yeah, will do that as its beginning to bloat slightly